src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner/DashboardTopBanner.module.scss

```diff
@@ -18,6 +18,9 @@
   }
 
   .chart {
+    display: inline-flex;
```

Comment:
Any reason this and `.chartContainer` need to be `inline-` elements? Intuitively I feel they're supposed to act more like block-level containers?

Reply:
I thought they needed to for a reason that I can’t reproduce so I guess [let’s go for flex](https://github.com/mozilla/blurts-server/pull/3570/commits/51799ca05367c9759eaec89e08e63b8b437f0dda).